client/Containers/Login.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+class Login extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
+
+  //componentDidUpdate
+  //perform axios call to authenticate user
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(this.state);
+    this.setState({ email: '', password: '' });
+  };
+
+  handleChange = (e) => {
+    this.setState({ [e.target.name]: e.target.value });
+  };
```

Comment:
I think this might cause state to only store the email or the password, depending on which was last changed. I think changing it like this should avoid this (along with some optional destructuring of `name` and `value`):
```javascript
handleChange = ({target: { name, value } }) => {
  this.setState({ ...state, [name]: [value] });
}
```

Reply:
updated